docs/requirements/path-unwinding-forwarding-requirements.md

```diff
@@ -0,0 +1,99 @@
+
+For examples, a user has stATOM on Osmosis, they want to move the stATOM to Evmos, instead of going from Osmosis --> Stride --> Evmos, a user tries to unwind themself and routes the tokens Osmosis --> Cosmos Hub --> Evmos. 
+
+### 3. Moving a token that originated from an interoperability zone or chain, or asset issuer
+
+Tokens that originate from other blockchain ecosystems that don't yet support IBC, flow into the Cosmos ecosystem through interoperability zones. These tokens are then sent onto other chains with a specific use case for these tokens and a user could want to move this token from one chain to another. 
+
+For example, ETH from Ethereum flows into Osmosis via Axelar, where the final step moving ETH from Ethereum to Osmosis uses an ICS-20 transfer from Axelar to Osmosis. A user may then want to move ETH from Osmosis onto another chain, for example Injective. However, the path with 1 hop would be a transfer from Axelar to Injective.
+
+### 4. Moving a token from one chain to another, swapping the token and transferring it onwards to a new destination
+
+A user on one chain, for example the Cosmos Hub holds an asset, e.g. ATOM and wants to instead have AKT on Akash. The user must transfer to ATOM to a DEX chain, swap the ATOM for AKT and then send the AKT onwards to Akash. 
+
+# Functional requirements
+
+## Assumptions and dependencies
+
+1. A functional relayer implementation is required for this feature.
+2. Routing information for the final hop for unwinding or for forwarding is configured for the user through a front end client, or configured by another means using off-chain data. 
+3. The feature will have no impact on the existing function of a typical ICS-20 transfer where a native token is sent to another chain.
+4. Fees will be treated the same as with other IBC applications.
+5. The functionality to enable a specific action before forwarding is not in scope of these requirements.
+6. If a transfer contains multiple unique tokens, the unwinding functionality only needs to support forwarding through the same path, i.e. if there is a transfer containing a native token and 1 hop denom being sent from source to destination, both tokens would unwind through the same path. In the future, it may be desirable for different tokens to be unwound through different paths, to support actions such as atomic transfer and supply liquidity to a liquidity pool, but it is currently out of scope for the first version of this feature. 
+
+## Features
+
+| ID | Description | Verification | Status | 
+| -- | ----------- | ------------ | ------ | 
+| 1.01 | When a user initiates a transfer to a destination chain with an IBC denom with > 1 hop, the token shall be sent back to its originating chain before being sent onto the destination as a default | | `Draft` | 
```

Comment:
We cannot do this as a default without having the user specify the destination chain port/channel which makes it opt in

Reply:
modified requirement